pkgs/tools/backup/tsm-client/default.nix

```diff
@@ -0,0 +1,125 @@
+{ lib
+, stdenv
+, fetchurl
+, gnutar
+, jdk  # optional (needed for `dsmj` GUI application)
```

Comment:
Good point! The documentation https://www-01.ibm.com/support/docview.wss?uid=swg21052223#Version%208.1 asks for "JRE 7 or JRE 8", so I'm now using "jdk8".

For certain functionality, it also recommends "the acl package" and "libdevmapper.so v1.01 or higher".

* Grepping through the result of the build process and using the `strings` utility shows that there are indeed some `setfacl` and `getfacl` command lines in binary files. While I never tested/used any acl functionality of TSM, I guess it's wiser to put `{s,g}etfacl` in the `PATH`. The increase in closure size should be minimal and it might help other users.
* The only file referencing `libdevmapper.so` seems to be `opt/tivoli/tsm/client/ba/bin/plugins/libPiIMG.so` (apart from many language-specific help files). However, autopatchelf doesn't complain about a missing library. Adding `lvm2` to `builtInputs` also does not generate a dependency on `lvm2`. So I'm ignoring this (optional) dependency for now.